cluster/gce/gci/configure-helper.sh

```diff
@@ -626,6 +635,8 @@ function prepare-etcd-manifest {
   sed -i -e "s@{{ *etcd_cluster *}}@$etcd_cluster@g" "${temp_file}"
   sed -i -e "s@{{ *storage_backend *}}@${STORAGE_BACKEND:-}@g" "${temp_file}"
   sed -i -e "s@{{ *cluster_state *}}@$cluster_state@g" "${temp_file}"
+  sed -i -e "s@{{ *etcd_protocol *}}@https@g" "${temp_file}"
```

Comment:
We want always to secure peer communication inside etcd cluster by SSL, to prevent malicious users from changing etcd state. It shouldn't have any performance impact in case of a single node clusters, as SSL is used only for communication between peers (so, it will only affect HA deployments).